src/ray/util/util_test.cc

```diff
@@ -48,7 +48,7 @@ TEST(UtilTest, ParseCommandLineTest) {
     ASSERT_EQ(ParseCommandLine(R"(\" )", syn), ArgList({R"(")"}));
     ASSERT_EQ(ParseCommandLine(R"(" a")", syn), ArgList({R"( a)"}));
     ASSERT_EQ(ParseCommandLine(R"("\\")", syn), ArgList({R"(\)"}));
-    ASSERT_EQ(ParseCommandLine(R"("\"")", syn), ArgList({R"(")"}));
+    ASSERT_EQ(ParseCommandLine("\"\\\"\"", syn), ArgList({R"(")"}));
```

Comment:
can you add the rawstring version as a comment?  something like 
```suggestion
    ASSERT_EQ(ParseCommandLine(/*R"("\"")"*/ "\"\\\"\"", syn), ArgList({R"(")"}));
```